### ISSUE TITLE
Some Help

### ISSUE THREAD

**Issue by MarkSimonovic** [manuparra/phpDIServer#2]
Hey Manu how are you, I'm in a project right know and I want to use your Di Server Configuration, can you help me out? We can talk about payment if you want to.

**Comment by dlgoodchild** [comment]
I don't think this project is maintained or that it works, the content includes lots of "tempuri.com".

**Comment by manuparra** [comment]
Hi, yes this should work perfectly for SAP BO DiServer. Using tempuri.com, it is just a template for the URL of the services that are deployed and you can adapt it to your own domain or network. Keep in mind that this has been used to provide access to SAP BO DiServer BackEnd functionality via SOAP and every environment is different, that's why I use a dummy URL as and services endpoint. Best regards.

**Comment by manuparra** [comment]
> Hey Manu how are you, I'm in a project right know and I want to use your Di Server Configuration, can you help me out? We can talk about payment if you want to.

Did you solve your problem ?
